README.md

```diff
@@ -17,14 +17,20 @@ A FIDO-compliant web app that has biometric authentication capabilities. Built u
 
 ## 🗺 Project Layout
 
-- [`fido_app/`](/fido_app): The Flask web app, which handles HTTP request processing and frontend-database interaction
-- [`templates/`](/fido_app/templates): The HTML files for the login, registration, and profile pages
-- [`static/`](/fido_app/static): Assets for our webpages, such as images, JS scripts, and CSS stylesheets
-- [`run.py`](run.py): The driving code for the server (just call `python run.py`)
+- [`web`](/web): The Flask Web App
+  - [`fido_app/`](/web/fido_app): The Flask web app, which handles HTTP request processing and frontend-database interaction
+  - [`templates/`](/web/fido_app/templates): The HTML files for the login, registration, and profile pages
+  - [`static/`](/web/fido_app/static): Assets for our webpages, such as images, JS scripts, and CSS stylesheets
+  - [`app.py`](/web/app.py): The driving code for the server (just call `python app.py`)
+- [`docker-compose.yml`](docker-compose.yml): Orchestration for the flask app + MariaDB server
+- [`Caddyfile`](Caddyfile): Reverse HTTPS proxy (see [Caddy's webpage](https://caddyserver.com/))
+- [`fido_app.service`](fido_app.service): Systemd service for starting the web server using `docker-compose` on system startup (used on production servers)
+- [`.github/workflows`](.github/workflows): GitHub Actions workflow definitions
 
 ## 🚀 Quick Start
 
 ```bash
+cd web
 pip install -r requirements.txt # Downloads all required Python packages
 python setup-db.py              # Creates a test database
 python run.py                   # Runs the server in development mode
```

Comment:
I'm guessing this is supposed to be `python wsgi.py` to be consistent with the edits in CONTRIBUTING.md, since `run.py` doesn't exist anymore.

Reply:
Thanks for pointing this out. There were also a few other inconsistencies with the `README.md` file that I fixed in 5476cd0.